chain/client/src/info.rs

```diff
@@ -208,20 +208,20 @@ fn display_sync_status(
         SyncStatus::NoSync => format!("#{:>8} {:>44}", head.height, head.last_block_hash),
         SyncStatus::HeaderSync { current_height, highest_height } => {
             let percent = if *highest_height <= genesis_height {
-                0
+                0.0
             } else {
-                (min(current_height, highest_height) - genesis_height) * 100
-                    / (highest_height - genesis_height)
+                (((min(current_height, highest_height) - genesis_height) * 100) as f64)
+                    / ((highest_height - genesis_height) as f64)
             };
-            format!("#{:>8} Downloading headers {}%", head.height, percent)
+            format!("#{:>8} Downloading headers {:.5}% ({})", head.height, percent, highest_height - current_height)
```

Comment:
5 digits seems too many. What about 2?

Reply:
Originally I set it to 5, so you would a different if the number of blocks changes by 1. That may be too much, if we change the limit to 3 we would see the different between two reports. For example:
```
Dec 13 23:54:38.081  INFO stats: #28102841 Downloading blocks 98.13299% (59492)  19/18/40 peers ⬇ 231.4kiB/s ⬆ 39.6kiB/s 3.40 bps 1.15 Tgas/s CPU: 21%, Mem: 2.0 GiB    
Dec 13 23:54:48.086  INFO stats: #28102891 Downloading blocks 98.13425% (59452)  19/18/40 peers ⬇ 240.4kiB/s ⬆ 41.4kiB/s 3.90 bps 1.02 Tgas/s CPU: 21%, Mem: 2.0 GiB
```